.gitlab-ci.yml

```diff
@@ -192,14 +194,56 @@ build-windows:
 
 publish-docker:
   stage:                           publish
-  only:                            *releaseable_branches
-  cache:                           {}
   dependencies:
     - build-linux
   tags:
-    - shell
+    - kubernetes-parity-build
+  environment:
+    name: parity-build
+  cache:                           {}
+  only:                            *releaseable_branches
+  except:
+    - master
```

Comment:
This is supposed to mean it shouldn't run on master to not to publish nightly images.

Reply:
releasable_branches contains no master - so it will already not run on commits to the master branch.

Reply:
changed to avoid just `nightly` schedule